### PR TITLE
Seems like it's better for users to provide their own mapbox key afte…

### PR DESCRIPTION
**Pull request by SmilyOrg**
…r all, since mine went over the limit pretty quickly

### CODE REVIEW

File: docs/examples/ModestMapsExample/src/ModestMapsExample.ls [hunk 1]
```diff
@@ -48,8 +48,8 @@ package
             //stage.reportFps = true;
             
             //create some providers
-            _mapProviders.pushSingle(new MapboxProvider(File.loadTextFile("assets/mapbox/id.txt"), File.loadTextFile("assets/mapbox/token.txt")));
             _mapProviders.pushSingle(new MicrosoftRoadMapProvider(true));
+            //_mapProviders.pushSingle(new MapboxProvider(File.loadTextFile("assets/mapbox/id.txt"), File.loadTextFile("assets/mapbox/token.txt")));
```
Comment (bengarney) [inline docs/examples/ModestMapsExample/src/ModestMapsExample.ls]: Can you add something that loads the id, then prints a big warning/error if you haven't changed it? And obviously only loads the layer if so.

**Comment by bengarney** [comment]
One nit!

File: docs/examples/ModestMapsExample/src/ModestMapsExample.ls (hunk 1 above)
Comment (SmilyOrg) [inline docs/examples/ModestMapsExample/src/ModestMapsExample.ls]: Good idea!

**Comment by SmilyOrg** [comment]
Now if you try to use the mapbox provider without changing anything it prints `Mapbox access token invalid, please check for spaces and length: Replace the contents of this file (assets/mapbox/token.txt) with your Mapbox access token!`

**Pull request merged**